### PR TITLE
[one-cmds] Support boolean type in the command schema

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit support boolean type in the command schema.

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/one-cmds/dummy-driver/src/dummy-compiler.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * dummy-infer only tests its interface rather than its functionality.
```
Comment (seanshpark) [inline compiler/one-cmds/dummy-driver/src/dummy-compiler.cpp:18]: `dummy-infer` -> `dummy-compiler` ?

File: compiler/one-cmds/dummy-driver/CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # dummy driver for interface test
+set(DUMMY_COMPILER_SRC src/dummy-compiler.cpp)
```
Comment (seanshpark) [inline compiler/one-cmds/dummy-driver/CMakeLists.txt:2]: maybe add a comment to describe diff of `compiler.cpp` vs `compile.cpp` ?

**Review by shs-park — APPROVED** [review]
LGTM!
=)

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Comment by mhs4670go** [comment]
@seanshpark Merge please:)

**Pull request merged**